tools/common/StaticRegistrar.cs

```diff
@@ -689,7 +689,7 @@ protected override bool HasThisAttribute (MethodDefinition method)
 			get {
 				if (IsSingleAssembly)
 					return App.Is64Build;
-				return Target.Is64Build;
+				return Target != null ? Target.Is64Build : App.Is64Build;
```

Comment:
it's not clear why `Target` would be `null` because of the other changes ?

Reply:
It's null now because it's no longer a single assembly build. And the mmp tool does not initialize the registrar via the `Target` constructor, but via the `App` one.

Reply:
Adding comment in the code.